tox.ini

```diff
@@ -68,10 +68,10 @@ commands =
 deps =
     check-manifest
     twine
+    build
 commands =
     check-manifest
-    python3 setup.py check -m -s
```

Comment:
I'm not aware of any full replacement for `python setup.py check` - `twine check` is the best we've got but it's not a drop-in solution.